### PR TITLE
Update colors for bootstrap

### PR DESCRIPTION
**Pull request by summer-cook**
# Summary

- This configures the webstore to define custom default colors/override bootstrap's defaults to be grayscale.
- It also sets up one place- `theme.scss` where future instances can change these default colors to match their branding. this is also the place where any other bootstrap default variables should be added/overridden
- It also configures the previously created colors.js file to use these same variables so that we will not lose any functionality/everything looks visually the same after this is merged.

# Related
https://github.com/scientist-softserv/webstore/issues/65

Component Library PR (must be merged for this to work): https://github.com/scientist-softserv/webstore-component-library/pull/67

# Acceptance Criteria
- [ ] default variables are set to grayscale

### CODE REVIEW

File: utils/theme/colors.js [hunk 1]
```diff
@@ -1,11 +1,15 @@
+// import "../../node_modules/bootstrap/scss/_functions.scss"
+// import "../../node_modules/bootstrap/scss/_variables.scss"
```
Comment (alishaevn) [inline utils/theme/colors.js:2]:
```suggestion
```

File: styles/globals.scss [hunk 2]
```diff
@@ -0,0 +1,15 @@
+// import custom theme variables to override any bootstrap defaults
```
Comment (alishaevn) [inline styles/globals.scss:1]: can we move this file into "utils/theme"? the goal is that all files that need/can be updated per webstore instance be inside the "utils" folder. for the default layout anyway.

File: styles/theme.scss [hunk 3]
```diff
@@ -0,0 +1,25 @@
+// these colors can be customized for each webstore instance. they are the default bootstrap theme colors and are used in bootstrap's theme_colors sass map, so updating these will also make sure that text overlaying areas with these background colors remains accessible. See https://getbootstrap.com/docs/5.0/customize/sass/#maps-and-loops for more details on how to add additional theme colors to this map.
```
Comment (alishaevn) [inline styles/theme.scss:1]: can we move this file into "utils/theme"? remember to export it from "theme/index" if it's needed outside the "theme" folder.

File: styles/globals.scss (hunk 2 above)
Comment (alishaevn) [reply styles/globals.scss:1]: remember to export it from "theme/index" if it's needed outside the "theme" folder.

File: styles/theme.scss (hunk 3 above)
Comment (alishaevn) [reply styles/theme.scss:1]: actually....it feels like we don't need these colors defined here AND in colors.js?? right? 

what do you think about putting this code into colors.js in place of lines 7-12 and deleting this file?

**Review by alishaevn — CHANGES_REQUESTED** [review]

File: utils/theme/globals.scss [hunk 4]
```diff
@@ -0,0 +1,15 @@
+// import custom theme variables to override any bootstrap defaults
+@import './theme.module.scss';
+
+// bootstrap must be imported after any variable overrides so that your custom variable settings will be pulled into BS's classes.
+@import "~bootstrap/scss/bootstrap";
```
Comment (alishaevn) [inline utils/theme/globals.scss:5]:
```suggestion
@import '~bootstrap/scss/bootstrap';
```

File: utils/theme/theme.module.scss [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+$success: #666666; 
+$info: #999999; 
+$warning: #2a2a2a;
+$danger: #606060;
+$light: #f8f9fa;
+$dark: #212529;
+$white: #ffffff;
+
+
+// these variables are also exported for direct use in components if needed, or in colors.js
```
Comment (alishaevn) [inline utils/theme/theme.module.scss:14]: oh yeah, can we update this to mention importing these colors into variables.js and _not_ directly into components?

**Review by alishaevn — CHANGES_REQUESTED** [review]
2 more things I forgot to address while we were pairing. 😅

**Comment by summer-cook** [comment]
@alishaevn just resolved your other 2 suggestions

**Review by alishaevn — APPROVED** [review]

**Pull request merged**